app/routers/user.py

```diff
@@ -42,4 +42,11 @@ def delete_user(user_id: int):
     if user == None:
         raise HTTPException(status_code=404, detail=f"A user with id {user_id} was not found")
     user_service.deleteUser(user_id)
-    return user
\ No newline at end of file
+    return user
+
+@router.put("/user/{user_id}", tags=["user"])
```

Comment:
this endpoint is not related with the PR. In the future this type of code should be done in a separated PR.

Reply:
this comment is not a blocker.